aggregator/apps/api/v1/stitcher.py

```diff
@@ -96,9 +106,10 @@ def get_cancelled_ballot_details(self) -> List[Dict]:
         """
         cancelled_ballot_details = []
         for ballot in self.cancelled_ballots:
+            reason_metadata = get_ballot_cancellation_reason_metadata(ballot)
             cancelled_ballot = {
                 "ballot_paper_id": ballot["ballot_paper_id"],
-                "detail": get_ballot_cancellation_reason(ballot),
+                "detail": reason_metadata["detail"] or None,
```

Comment:
I think this will throw an exception if the key doesn't exist rather than continuing with a value of `None`

Reply:
Yep I think you're right. I've pushed a fixup commit that I think addresses this?